efinpro/Controllers/HomeController.cs

```diff
@@ -14,6 +15,8 @@ public class HomeController : EFinProBaseController
         Users _users = null;
```

Comment:
provide access specifiers for all call member data

Reply:
Access specifiers for all call member data has provided.